operators/hpe-csi-operator/licenses/LICENSE

```diff
@@ -0,0 +1,192 @@
+Apache License
```

Comment:
Is this standard as part of operators too?

Reply:
I have looked at others at operatorhub.io/ and doesn't seem like this is required. Since we have a License file anyway for this repo, will remove this.